src/ORM/Connect/PDOConnector.php

```diff
@@ -395,17 +405,20 @@ public function preparedQuery($sql, $parameters, $errorLevel = E_USER_ERROR)
     {
         $this->beforeQuery($sql);
 
-        // Prepare statement
+        // Fetch cached statement, or create it
         $statement = $this->getOrPrepareStatement($sql);
```

Comment:
Sorry to be a pain, can you rename to `$statementHandle`?

Reply:
not a pain, that's what peer review is for ;-)